docs/tutorials/pre_executed/zubercal-ps1-snad.ipynb

```diff
@@ -0,0 +1,2344 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    snad_ps1 = snad_catalog.crossmatch(
</pre><p>you can provide <code>require_right_margin=False</code>,and the warnings won't show up.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/444/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>It looks like we don't have it for nested_join...</p>